src/uct/ib/ud/verbs/ud_verbs.c

```diff
@@ -743,6 +743,14 @@ uct_ud_verbs_query_tl_devices(uct_md_h md,
                                      num_tl_devices_p);
 }
 
-UCT_TL_DEFINE(&uct_ib_component, ud_verbs, uct_ud_verbs_query_tl_devices,
-              uct_ud_verbs_iface_t,  "UD_VERBS_",
-              uct_ud_verbs_iface_config_table, uct_ud_iface_config_t);
+UCT_TL_REGISTER_DEF(&uct_ib_component, ud_verbs, uct_ud_verbs_query_tl_devices,
+                    uct_ud_verbs_iface_t,  "UD_VERBS_",
+                    uct_ud_verbs_iface_config_table, uct_ud_iface_config_t);
+
+UCT_IB_TL_INIT(ud_verbs)
+{
+}
+
+UCT_IB_TL_CLEANUP(ud_verbs)
+{
+}
```

Comment:
looks like this sequence is repeated for all the macros. Can it be hidden by a common macro?

Reply:
> looks like this sequence is repeated for all the macros. Can it be hidden by a common macro?

yep, why not